zerver/lib/actions.py

```diff
@@ -3332,6 +3332,14 @@ def do_delete_message(user_profile, message):
     move_message_to_archive(message.id)
     send_event(event, ums)
 
+def do_update_attachment_list(user_profile, op, attachment):
+    # type: (UserProfile, str, Attachment) -> None
+    event = {
+        'type': 'attachments',
+        'op': op,
+        'attachment': attachment}
+    send_event(event, [user_profile.id])
```

Comment:
Maybe call this `notify_attachment_update`?  `do_foo` names are for things that actually modify the database.